cmd/antrea-controller/options_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022 Antrea Authors
```

Comment:
@wenqiq could you update 2022 to 2023 for all new files. Thanks.

Reply:
done